components/CramerView/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <!-- vim: set tabstop=4 softtabstop=0 expandtab shiftwidth=4 smarttab : -->
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.tibco.ep.sp.contrib</groupId>
+    <artifactId>CramerView</artifactId>
+    <packaging>ep-liveview-fragment</packaging>
+    <version>1.0.0</version>
+    <name>CramerView</name>
```

Comment:
Should be a human readable name like 'Cramer View'

Reply:
ok